### ISSUE TITLE
Problem grabbing clip of video (invalid handle)

### ISSUE THREAD

**Issue by ScriptKittyGardian** [Zulko/moviepy#569]
I'm working on a script to download and edit videos from youtube, but I keep gettting this error:
```
Traceback (most recent call last):
  File "C:\Python34\YoutubeProject\stageOne.py", line 56, in <module>
    clip.write_videofile(os.path.dirname(os.path.realpath(__file__)) + '/' + requestDate + '/' + str(i) + "video" + '.mp4')
  File "<decorator-gen-51>", line 2, in write_videofile
  File "C:\Python34\lib\site-packages\moviepy\decorators.py", line 54, in requires_duration
    return f(clip, *a, **k)
  File "<decorator-gen-50>", line 2, in write_videofile
  File "C:\Python34\lib\site-packages\moviepy\decorators.py", line 137, in use_clip_fps_by_default
    return f(clip, *new_a, **new_kw)
  File "<decorator-gen-49>", line 2, in write_videofile
  File "C:\Python34\lib\site-packages\moviepy\decorators.py", line 22, in convert_masks_to_RGB
    return f(clip, *a, **k)
  File "C:\Python34\lib\site-packages\moviepy\video\VideoClip.py", line 349, in write_videofile
    progress_bar=progress_bar)
  File "C:\Python34\lib\site-packages\moviepy\video\io\ffmpeg_writer.py", line 204, in ffmpeg_write_video
    ffmpeg_params=ffmpeg_params)
  File "C:\Python34\lib\site-packages\moviepy\video\io\ffmpeg_writer.py", line 127, in __init__
    self.proc = sp.Popen(cmd, **popen_params)
  File "C:\Python34\lib\subprocess.py", line 754, in __init__
    _cleanup()
  File "C:\Python34\lib\subprocess.py", line 474, in _cleanup
    res = inst._internal_poll(_deadstate=sys.maxsize)
  File "C:\Python34\lib\subprocess.py", line 1147, in _internal_poll
    if _WaitForSingleObject(self._handle, 0) == _WAIT_OBJECT_0:
OSError: [WinError 6] The handle is invalid
```

Whenever I try to open it as a video clip.  The other files work fine.  I've tried adding a delay after the download completes, but that doesn't change anything.  Apparently I can't upload the video here.

**Comment by Gloin1313** [comment]
Does this help? https://stackoverflow.com/questions/43966523/getting-oserror-winerror-6-the-handle-is-invalid-in-videofileclip-function

**Comment by ScriptKittyGardian** [comment]
@Gloin1313 never really looked like it got resolved.  My program is designed to process thousands of videos autonomously over a few hours, so restarting every 2 or 3 videos in order to flush out the bad handle seems kind of silly.

**Comment by tburrows13** [comment]
Should be fixed in v1.0.3.
